.cspell/cspell.json

```diff
@@ -78,6 +78,13 @@
       "filename": "/core/modules/simpletest/tests/file.test",
       "ignoreWords": ["NONEXISTINGFILENAME", "ufail"]
     },
+    {
+      "filename": "/core/modules/user/user.module",
+      "ignoreWords": [
+        "abcdefghijkmnopqrstuvwxyz", "ABCDEFGHJKLMNPQRSTUVWXYZ",
+        "plid", "23456789"
+      ]
+    },
```

Comment:
I think these exceptions can be removed because we used an inline disable rule instead:

```
  // cspell:disable-next-line
  $allowable_characters = 'abcdefghijkmnopqrstuvwxyzABCDEFGHJKLMNPQRSTUVWXYZ23456789';
```

`plid` was separately added to the dictionary. See 3817a40b1bcd43ee786587c8c86a0e586471cac0 and  2bacdd1aa3ad5145170afbb8f31af9c2c8be30f7

```suggestion
```